### PR TITLE
📖 [Docs] Document tilt-support install

### PR DESCRIPTION
**Pull request by bentito**
# Description

## Reviewer Checklist

- [ ] API Go Documentation
- [ ] Tests: Unit Tests (and E2E Tests, if appropriate)
- [ ] Comprehensive Commit Messages
- [ ] Links to related GitHub Issue(s)

### CODE REVIEW

File: tilt.md [hunk 1]
```diff
@@ -32,6 +32,15 @@ operator-controller requires
 their own Tiltfiles, before proceeding. If you want to use Tilt, make sure you specify a unique `--port` flag to each
 `tilt up` invocation.
 
+## Install tilt-support Repo, one level up
+You must install the tilt-support repo one level up from this repo
```
Comment (LalatenduMohanty) [inline tilt.md:36]: I do not think this text `one level up from this repo` is adding any extra value as the below command clearly mentions which repo to clone.

**Review by LalatenduMohanty — CHANGES_REQUESTED** [review]

**Comment by LalatenduMohanty** [comment]
Can we add a link of this document in https://github.com/operator-framework/operator-controller/blob/main/docs/drafts/developer.md? As both of these docs should be at least linked if not consolidated.

**Comment by bentito** [comment]
> Can we add a link of this document in https://github.com/operator-framework/operator-controller/blob/main/docs/drafts/developer.md? As both of these docs should be at least linked if not consolidated.

I think the idea to simply merging this text about tilt into a section of the developer.md doc is a good one. I'd like to just finish up this PR so we have better text to merge. We can do the tilt -> developer merge in a follow up PR

**Comment by bentito** [comment]
Created this issue to track combining tilt.md into developer.md: https://github.com/operator-framework/operator-controller/issues/1242

**Review by joelanford — APPROVED** [review]

**Pull request merged**